File: beecrowd_urionlinejudge/uri1010CalculoSimples.cpp
#include <iostream>
#include<iomanip>
#include <stdio.h>
using namespace std;

int main() {

int CodigoPeca1,NumeroPecas1,CodigoPeca2,NumeroPecas2;
float ValorUnitarioPecas1,ValorUnitarioPecas2,ValorPagar;

cin >>CodigoPeca1 >>NumeroPecas1 >>ValorUnitarioPecas1;
cin >>CodigoPeca2 >>NumeroPecas2 >>ValorUnitarioPecas2;

ValorPagar = NumeroPecas1*ValorUnitarioPecas1 + NumeroPecas2*ValorUnitarioPecas2;

printf("VALOR A PAGAR: R$ %.2f\n",ValorPagar);


    
	return 0;
}

File: beecrowd_urionlinejudge/uri1010CalculoSimples.pas
program main;

var CodigoPeca1,NumeroPecas1,CodigoPeca2,NumeroPecas2 : integer;
var ValorUnitarioPecas1,ValorUnitarioPecas2,ValorPagar : real;

Begin

   read(CodigoPeca1);
   read(NumeroPecas1);
   read(ValorUnitarioPecas1);
   
   read(CodigoPeca2);
   read(NumeroPecas2);
   read(ValorUnitarioPecas2);
   
ValorPagar := NumeroPecas1*ValorUnitarioPecas1 + NumeroPecas2*ValorUnitarioPecas2;


   writeln('VALOR A PAGAR: R$ ',ValorPagar:1:2);
end.
